packages/bot-web-ui/src/components/quick-strategy/quick-strategy.jsx

```diff
@@ -57,6 +57,11 @@ const QuickStrategyForm = ({
             const validation_errors = validateQuickStrategy(values);
             const is_valid = Object.keys(validation_errors).length === 0;
             const is_submit_enabled = !isSubmitting && is_valid;
+            const is_safari =
+                /constructor/i.test(window.HTMLElement) ||
+                (function (p) {
+                    return p.toString() === '[object SafariRemoteNotification]';
+                })(!window.safari || (typeof safari !== 'undefined' && window.safari.pushNotification));
```

Comment:
it's a good idea if you move is_saffari into share packages 
e.g /deriv-app/packages/shared/src/utils/os/browser_detect-.js

Reply:
sure, will do!

Reply:
changed, check pls